### DISCUSSION TITLE
Add cdnURLs for PersianBlocker

### DISCUSSION THREAD

**Discussion by MasterKia** [uBlockOrigin/uAssets#17921] [General] [answered]
As it currently stands, the PersianBlocker list has no cdnURLs:
https://github.com/gorhill/uBlock/blob/42106cf5ff390304e0bebc63f03a7c109bd84106/assets/assets.json#L493

So I propose:

contentURL:
`https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlocker.txt`

cdnURLs:
`https://cdn.statically.io/gh/MasterKia/PersianBlocker/main/PersianBlocker.txt`

`https://cdn.jsdelivr.net/gh/MasterKia/PersianBlocker@main/PersianBlocker.txt`

@gorhill

**Comment by krystian3w** [comment]
For now unpopular: [![](https://data.jsdelivr.com/v1/package/gh/MasterKia/PersianBlocker/badge)](https://www.jsdelivr.com/package/gh/MasterKia/PersianBlocker?tab=stats)

**Comment by ghost** [comment, marked as answer]
https://github.com/gorhill/uBlock/commit/c1ac09b96ae23a5e29486e73e1f520eead1739bc

**Reply by MasterKia** [reply]
@gorhill Now `contentURL` has a duplicated entry.

Also `statically.io` is in both `contentURL` and `cdnURLs`. I think it's meant to be only in the latter.

**Reply by krystian3w** [reply]
Also Dutch boys are ignored due Netherlands down in first semi final of Eurovision 2023?

https://github.com/uBlockOrigin/uAssets/discussions/17919#discussioncomment-5800673
